test/classes/vass/var_x_=_nil.chpl

```diff
@@ -0,0 +1,22 @@
+// test for GitHub issue #19244
+// this used to crash during --no-local compilations
+// this issue becomes moot if we accept the proposal in #19672
+
+// code from #19244
+use CTypes;
+var ptr = c_nil;
+
+// additional code
+
+var x = nil;
+writeln(x);
+writeln(ptr);
+
+record RR {
+  var value = nil;
+}
+
+var rr: RR;
+__primitive("=", rr.value, nil); // this happens upon "rr.value = nil;"
```

Comment:
Why is this using the primitive instead of `rr.value = nil` itself?

Reply:
There are two overloads of `operator =` in ChapelBase.chpl that match `nil = nil;` -- one for borrowed nilable classes and the other for unmanaged nilable classes. So writing `rr.value = nil;` gave me an "ambiguous call" error. Inlining the body of either overload allowed the test to compile.

Reply:
Huh.  That's unfortunate, but makes sense